ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -41,15 +39,6 @@
   public ReactActivityDelegate(Activity activity, @Nullable String mainComponentName) {
     mActivity = activity;
     mMainComponentName = mainComponentName;
-    mFragmentActivity = null;
-  }
-
-  public ReactActivityDelegate(
```

Comment:
`ReactActivityDelegate ` is a public method that might be used by product code in open source. As this is a breaking change, let's hold off on removing it. Can you mark it as deprecated instead? We can then remove it in the release following the one where this deprecation notice is sent out.